benefits/eligibility/analytics.py

```diff
@@ -25,8 +25,11 @@ class ReturnedEligibilityEvent(EligibilityEvent):
 
     def __init__(self, request, eligibility_types, status, error=None):
         super().__init__(request, "returned eligibility", eligibility_types)
-        if str(status).lower() in ("error", "fail", "success"):
+        status = str(status).lower()
+        if status in ("error", "fail", "success"):
             self.update_event_properties(status=status, error=error)
+        if status == "success":
+            self.update_user_properties(eligibility_types=eligibility_types)
```

Comment:
Is the thought here that we want to keep track of the user's most recently confirmed eligibility type? (I just read through [User-properties-and-event-properties](https://help.amplitude.com/hc/en-us/articles/115002380567-User-properties-and-event-properties) to refresh on how user properties work)

Reply:
Good question!

My thinking with the user prop here is we may want to segment our users based on which eligibility they were verified for. Note this is different from counting the number of successful eligibility verifications, which is _event_ based and not _user_ based.

The event will also have a prop with the list of verified type(s), so we can always do that level of analysis as well.